railties/lib/rails/generators/rails/app/templates/package.json.tt

```diff
@@ -1,5 +1,10 @@
 {
   "name": "<%= app_name %>",
   "private": true,
-  "dependencies": {}
+  "dependencies": {
+    "rails-ujs": "<%= Rails::VERSION::STRING %>"<% unless options[:skip_turbolinks] %>,
```

Comment:
Hi there! I think there's something wrong here... `"rails-ujs" unless options[:skip_turbolinks]`, etc...

Reply:
This is correct: we’re skipping the *next* dependency.